tests/installation/npmTest.ts

```diff
@@ -75,7 +90,7 @@ type NPMTestFixtures = {
   _auto: void;
   _browsersPath: string;
   tmpWorkspace: string;
-  installedSoftwareOnDisk: () => Promise<string[]>;
+  browsers: { isolateBrowsers: boolean, browsersPath: string };
```

Comment:
Let's call it `installedBrowsers` or `installedExecutables` to disambiguate? but also in that case maybe old installedSoftwareOnDisk is just as good?

Reply:
Another idea is to use `checkInstalledSoftwareOnDisk(...)` fixture which would depend on `{ isolateBrowsers, _browsersPath }` and you won't have to pass browsers around.